cmd/watch-termination/main.go

```diff
@@ -253,6 +253,20 @@ func (w *terminationFileWriter) WriteToTerminationLog(bs []byte) (int, error) {
 	}
 
 	if w.logger == nil {
+		if exist, err := fileExists(w.fn); err != nil {
+			return 0, err
```

Comment:
chmod?

Reply:
it really only applies to new files.
already existing files are changed by the `Fixing audit permissions` container.

Reply:
which gets executed on every restart

Reply:
ack